packages/authgear-react-native/src/nativemodule.ts

```diff
@@ -148,7 +148,10 @@ export async function signWithDPoPPrivateKey(
 }
 
 export async function checkDPoPPrivateKey(kid: string): Promise<boolean> {
-  return _wrapPromise(AuthgearReactNative.checkDPoPPrivateKey({ kid }));
+  const result = await _wrapPromise(
+    AuthgearReactNative.checkDPoPPrivateKey({ kid })
+  );
+  return result === "true";
```

Comment:
Why do we need to do this? Can it be work around with returning a object with `ok: true | false`? Please document the reason when you are doing something non-trivial.

Reply:
https://github.com/authgear/authgear-sdk-js/blob/master/packages/authgear-react-native/src/nativemodule.ts#L5-L7

Maybe I also put this comment in the method.